eks-log-collector.sh

```diff
@@ -67,21 +84,23 @@ parse_options() {
   done
 }
 
-ok() {
-  echo "ok"
+ok()
+{
+  echo
 }
 
-info() {
+info()
+{
```

Comment:
For uniformity, lets leave this info() {

Reply:
Easy fix.  I do see there are many others where it does not follow ```() {```.  Do you want me to fix these as well?

Reply:
Fixed it all, and make  them uniform.